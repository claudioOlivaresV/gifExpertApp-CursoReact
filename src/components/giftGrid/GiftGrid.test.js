const { shallow } = require("enzyme");
const { GiftGrid } = require("./GiftGrid");

describe('Testing GifGrid', () => {
    const category = 'Goku'
    const wrapper = shallow(<GiftGrid category={category} />)
    test('should ', async () => {
        
        expect(wrapper).toMatchSnapshot()
    });
    
});