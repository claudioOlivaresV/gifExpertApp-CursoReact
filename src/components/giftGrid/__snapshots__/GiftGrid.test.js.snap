// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing GifGrid Probar que en snapshot sea igual al grid 1`] = `
<Fragment>
  <h3>
     
    Goku
  </h3>
  <WithStyles(LinearQuery) />
  <div
    className="makeStyles-root-1"
  >
    <WithStyles(ForwardRef(GridList))
      className="makeStyles-gridList-2"
      cols={2.5}
    />
  </div>
</Fragment>
`;
